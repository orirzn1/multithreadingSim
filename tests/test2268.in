L9
S6
O7
N11

T0
I@0x00000000
STORE $1, $1, 0x4
SUB $3, $5, $0
LOAD $1, $5, 0x4
LOAD $5, $4, $2
ADD $2, $5, $2
ADDI $5, $7, 0x4
HALT $0

T1
I@0x00000000
STORE $6, $1, $3
LOAD $6, $5, 0x2
ADDI $3, $3, 0x1
SUBI $7, $0, 0x4
SUBI $3, $7, 0x2
HALT $0

T2
I@0x00000000
ADD $1, $6, $5
SUBI $4, $2, 0x3
STORE $2, $2, $1
SUB $0, $4, $4
HALT $0

T3
I@0x00000000
SUBI $3, $4, 0x2
ADDI $3, $5, 0x4
LOAD $2, $0, 0x2
LOAD $1, $0, 0x4
ADD $3, $6, $0
LOAD $4, $1, 0x3
LOAD $1, $0, $2
ADD $7, $0, $7
ADDI $6, $2, 0x3
SUBI $7, $6, 0x1
HALT $0

T4
I@0x00000000
STORE $5, $3, $2
SUBI $0, $6, 0x1
SUB $4, $6, $6
SUBI $7, $0, 0x4
LOAD $5, $0, 0x3
STORE $4, $3, $6
LOAD $1, $1, $4
SUB $4, $7, $2
HALT $0

T5
I@0x00000000
ADDI $6, $0, 0x4
ADDI $5, $4, 0x1
SUB $3, $0, $6
ADDI $4, $1, 0x2
ADD $6, $7, $6
LOAD $0, $2, $7
SUBI $1, $1, 0x4
HALT $0

T6
I@0x00000000
LOAD $0, $4, 0x0
SUB $5, $4, $4
STORE $5, $5, 0x4
SUB $0, $7, $6
SUB $5, $6, $3
LOAD $1, $7, 0x3
HALT $0

T7
I@0x00000000
SUB $7, $0, $7
SUBI $1, $7, 0x2
SUB $7, $3, $6
LOAD $2, $3, 0x2
STORE $6, $2, $5
STORE $7, $7, $0
ADD $4, $4, $1
LOAD $3, $6, 0x0
HALT $0

T8
I@0x00000000
ADDI $4, $6, 0x2
LOAD $5, $2, 0x3
LOAD $5, $0, 0x1
LOAD $3, $7, 0x4
HALT $0

T9
I@0x00000000
SUB $1, $0, $3
LOAD $3, $5, $2
LOAD $7, $3, $6
ADD $5, $2, $0
SUB $4, $3, $5
ADDI $3, $3, 0x1
SUB $6, $7, $6
SUBI $5, $7, 0x3
HALT $0

T10
I@0x00000000
STORE $1, $7, $6
LOAD $2, $6, 0x3
SUB $3, $5, $3
ADD $5, $2, $5
LOAD $4, $5, $6
ADDI $7, $4, 0x1
SUBI $1, $1, 0x3
STORE $3, $1, $5
SUB $6, $1, $6
HALT $0

D@0x0
0x3
0x3
0x5
0x8
0x8
0x6
0xA
0x7
