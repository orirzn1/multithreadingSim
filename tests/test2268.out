
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFA	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFD	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x3	R7 = 0x2
Register file thread id 4:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFB	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFB
Register file thread id 5:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0x2	R5 = 0x1	R6 = 0x4	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 8:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 10:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x1
Blocked MT CPI for this program 4.337209

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFA	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFD	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x3	R7 = 0x2
Register file thread id 4:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFB	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFB
Register file thread id 5:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0x2	R5 = 0x1	R6 = 0x4	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0xFFFFFFFD	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 8:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0x0	R7 = 0x0
Register file thread id 10:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x1
Finegrained Multithreading CPI for this program 1.011628

